test/reporter-html/single-testid.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>QUnit Main Test Suite</title>
+	<link rel="stylesheet" href="../../dist/qunit.css">
+	<script src="../../dist/qunit.js"></script>
+  <script src="single-testid.js"></script>
```

Comment:
When you make a change based on a review, it helps if you immediately push that change, then the PR and diff are updated and I can see the result, instead of coming back to it later. That is, you say you're done, but I currently can't review your fix, since its still on your local machine (I assume).